rasa/utils/common.py

```diff
@@ -268,6 +268,7 @@ def filter(self, record: logging.LogRecord) -> bool:
         return False
 
 
+# TODO: This is now unused
```

Comment:
I can probably just drop this function but I'm not sure if this constitutes part of our public API and therefore needs to go through the whole deprecation process first

Reply:
No, you can safely drop it as it's just a `util` function outside of the `rasa.shared` package (everything within `rasa.shared` needs extra caution as it might be used by downstream dependencies such as Rasa X)